code/datums/brain_damage/special.dm

```diff
@@ -0,0 +1,150 @@
+//Brain traumas that are rare and/or somewhat beneficial;
+//they are the easiest to cure, which means that if you want
+//to keep them, you can't cure your other traumas
+/datum/brain_trauma/special
+
+#define OWNER 0
+#define STRANGER 1
+
+/datum/brain_trauma/special/split_personality
```

Comment:
This seems like it's going to get very, VERY griefy.

Reply:
may have to add some message to say that the ghost has to care for its own body and shit

Reply:
this is hillarious and I love it, it'll be fine, just add an admin log when people switch and make sure the ckeys are updated